package.json

```diff
@@ -27,7 +27,7 @@
     "scripts": [
       "build/**/*.js",
       "node_modules/balena-sdk/es2018/index.js",
-      "node_modules/balena-sync/build/sync/*.js",
+      "node_modules/balena-sync/build/**/*.js",
```

Comment:
I suspect (not fully confirmed) that the need for this change comes from this `balena-sync` PR:
https://github.com/balena-io-modules/balena-sync/pull/163

Curiously though, the error started manifesting with the removal of Capitano in CLI release v12.17.0.